### ISSUE TITLE
change NOBA to have 2 fleets?

### ISSUE THREAD

**Issue by gavinfay** [thefaylab/hydra_sim#19]
In looking at some preliminary model fits (!!!!) I noticed that there appear to be 2 distinct fishing patterns, in that most species don't have catches in the initial years except species 7.
With the fleet-wise specification of F in the model, this makes things a little screwy.
Suggest making the NOBA files to have 2 fleets, where fleet 1 is everything but species 7 and fleet 2 is species 7.
@sgaichas any thoughts?

(an aside, those initial fits do not appear to be capturing the apparent effects of recruitment variability seen in the index data - I have yet to look at the fits to the composition data, or indeed the comp data themselves to see if there are signals there too)

**Comment by sgaichas** [comment]
We can specify two fleets, and having flexibility with the number of fleets in general will be helpful for fitting the "real" data. 

If I understand correctly, we could just re-specify the fleet for species 7 as fleet 2 in the q definition and in the "fleet" column for time series without changing the input data time series in any other way, is that correct?

**Comment by gavinfay** [comment]
Yes, I think this is 'all' that is needed. I can make this change in my working copies but wanted to flag here so it could be incorporated into your autogen code.

**Comment by sgaichas** [comment]
@gavinfay I've just posted new input files (.dat, -ts.dat, .pin) that _should_ have the 2 fleets as described. Also they go to 10 length bins for each species. 

I've renamed ("_10bin") rather than written over the old ones because you have some changes in the .pin file that I haven't incorporated into the code yet. Also I know you were working with the fleet selectivities. 
 
So rather than break work in progress, we now have the ability to cut and paste (yay!) between new and old to get something I hope works.
